src/main/native/windows/process.cc

```diff
@@ -71,15 +71,15 @@ bool WaitableProcess::Create(const std::wstring& argv0,
                              LARGE_INTEGER* opt_out_start_time,
                              std::wstring* error) {
   return Create(argv0, argv_rest, env, wcwd, stdin_process, stdout_process,
-                stderr_process, opt_out_start_time, false, error);
+                stderr_process, opt_out_start_time, false, false, error);
```

Comment:
Do you know why changing this method affects `bazel run` command? I'm just a bit worried that this will cause some other place that we actually want to create a new process group to have a wrong behavior.

Reply:
This is one of two public Create overloads for WaitableProcess, and the method's comment mentions that it wants to inherit console. This specific overload is only called from blaze_util_windows.cc ExecuteProgram which in it's turn is called by ExecuteRunRequest, and ExecuteServerJvm.

ExecuteRunRequest is the method which we are interested in, so we could pass the boolean from there instead. However looking at blaze_util_posix.cc I think it's safe to say that ExecuteServerJvm is also expected to just "take-over" the current process by calling execv there.

Reply:
Sounds good, that's what I want to know! Thanks so much!